distribution/proxy/src/main/resources/bin/start.sh

```diff
@@ -134,6 +135,20 @@ if [ "$1" == "-v" ] || [ "$1" == "--version" ] ; then
     print_version
 fi
 
+
+function enable_agent() {
+    AGENT_PARAM="";
+    AGENT_FILE="${DEPLOY_DIR}/agent/shardingsphere-agent.jar"
+    if [ -f "$AGENT_FILE" ]; then
+      AGENT_PARAM=" -javaagent:${AGENT_FILE} "
+    fi
+}
+
+if [ "$1" == "-g" ] || [ "$1" == "--agent" ] ; then
```

Comment:
Is it possible that the `-g` is not in `$1`?

Reply:
> 
wow, thanks for your suggestion. I was doing that then.
this time it support random positon in parameters.